ionizer/ops.py

```diff
@@ -89,7 +92,9 @@ class GPI2(Operation):
     num_wires = 1
     num_params = 1
     ndim_params = (0,)
-
+    grad_method = "A"
+    parameter_frequencies = [(1,)]
+    
```

Comment:
Please define the `generator` method for the operation.

Reply:
In the docs and on the forum, i was told that i can define either generator, parameter_frequencies or grad_recipe.